spec/ics-2-consensus-requirements/README.md

```diff
@@ -0,0 +1,124 @@
+---
+ics: 2
+title: Consensus Verification
+stage: proposal
+category: ibc-core
+requires: 23
+required-by: 3
+author: Juwoon Yun <joon@tendermint.com>, Christopher Goes <cwgoes@tendermint.com>
+created: 2019-02-25
+modified: 2019-03-05
+---
+
+## Synopsis
+
+Consensus verification requires the properties that the chains on the network are 
+expected to satisfy. The properties are needed for efficient and safe verification on the 
+higher level mechanics, such as connection and channel semantics. The algorithm who uses these 
+properties to verify another chain is referred as "lightclient", which is embedded on the chains.
+
+## Specification
+
+### Motivation
+
+`Header` defines required properties of the blockchain on the network. The implementors can 
+check whether the consensus that they are using is qualified to be connected to the network or 
+not. If not, they can modify the algorithm or wrap it with additional logic to make it 
+compatible with the specification. It also provides base layer for the protocol that the other 
+components can rely on.
+
+### Desired Properties
+
```

Comment:
This should also talk about desired properties *of the specification* - namely that it cleanly abstracts over the kind of consensus algorithms which IBC can support by providing a minimal set of requirements for efficient & safe verification.